db/migrate/20230404092351_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.datetime :updated_at
+      t.datetime :created_at
+      t.integer :posts_counter
+
```

Comment:
- [ ] I think you have a problem here when I try to migrate it shows me this error in user table you can't define an already defined column 'created_at'. so kindly solve this probleem and make sure you have all the migration runing without problems to have the schema ready 🔥🤩



![image](https://user-images.githubusercontent.com/58553711/229794543-72aaf565-a371-406e-901a-ccba4006e3bc.png)